cfme/tests/test_modules_importable.py

```diff
@@ -0,0 +1,32 @@
+import py
+import pytest
+import cfme
+import subprocess
+import sys
+
+ROOT = py.path.local(cfme.__file__).dirpath()
+
+MODULES = sorted(x for x in ROOT.visit("*.py") if 'test_' not in x.basename)
+
+KNOWN_FAILURES = set(ROOT.dirpath().join(x) for x in[
```

Comment:
*QUESTION:* Would this list be better in a yaml file?

Reply:
these are supposed to be removed in a less critical followup, not something to be maintained over time